akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala

```diff
@@ -126,6 +126,7 @@ private[http] object OutgoingConnectionBlueprint {
           case (ctx, `terminationBackchannelInput`) ⇒
             ctx.finish()
             SameState
+          case (_, _) ⇒ SameState
```

Comment:
and this is alright? we don't silently ignore any programming mistakes? alternative is to throw IllegalStateException?